sdgym/summary.py

```diff
@@ -3,6 +3,8 @@
 import numpy as np
 import pandas as pd
 
+from sdgym.results import add_sheet
```

Comment:
If the `results.py` module is not being used anywhere else (which I think is not), I would copy whatever is necessary here and remove the module.